webpack/components/extensions/HostDetails/DetailsTabCards/HwPropertiesCard.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { propsToCamelCase } from 'foremanReact/common/helpers';
+import { translate as __ } from 'foremanReact/common/I18n';
+import {
+  DescriptionList,
+  DescriptionListTerm,
+  DescriptionListGroup,
+  DescriptionListDescription,
+  Text,
+  TextVariants,
+} from '@patternfly/react-core';
+import CardTemplate from 'foremanReact/components/HostDetails/Templates/CardItem/CardTemplate';
+
+const HostDisks = ({ totalDisks }) => {
+  if (!totalDisks) return null;
+  return (
+    <>
+      <DescriptionListTerm>{__('Storage')}</DescriptionListTerm>
+      <Text component={TextVariants.h4}>{totalDisks} {totalDisks > 1 ? __('disks') : __('disk')}</Text>
```

Comment:
I know we had trauma around this before, but this should be the simple use case:

```suggestion
      <Text component={TextVariants.h4}><TranslatedPlural count={totalDisks} singular={__('disk')} id="total-disks" /></Text>
```